subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include "hal/radio_df.h"
+#include "lll_df.h"
+
+/* @brief Function performs common steps for initialization and reset
+ * of Direction Finding LLL module.
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+static int init_reset(void);
+
+/* @brief Function performs Direction Finding initialization
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+int lll_df_init(void)
+{
+	int err;
+
+	err = radio_df_ant_configure();
+	if (err) {
+		return err;
+	}
+
+	err = init_reset();
+	if (err) {
+		return err;
+	}
+
+	return 0;
+}
+
+
+/* @brief Function performs Direction Finding reset
+ *
+ * @return	Zero in case of success, other value in case of failure.
+ */
+int lll_df_reset(void)
+{
+	int err;
+
+	err = init_reset();
+	if (err) {
+		return err;
+	}
+
+	return 0;
+}
+
+/* @brief Function provides number of available antennae.
+ *
+ * The number of antenna is hardware defined and it is provided via device tree.
+ *
+ * @return      Number of available antennae.
+ */
+uint8_t lll_df_ant_num_get(void)
+{
+	return radio_df_ant_num_get();
+}
+
+static int init_reset(void)
+{
+	return 0;
+}
```

Comment:
Shouldn't this be doing something?

Reply:
At the moment it is not doing anything.
It is left on @cvinayak request - to use the same modules template. Code will be removed during compilation.

Reply:
This is similar to above: https://github.com/zephyrproject-rtos/zephyr/pull/30262#discussion_r545686646